src/Surfnet/Stepup/Tests/Identity/Event/EventSerializationAndDeserializationTest.php

```diff
@@ -68,8 +71,17 @@ class EventSerializationAndDeserializationTest extends UnitTest
      */
     public function an_event_should_be_the_same_after_serialization_and_deserialization(SerializableInterface $event)
     {
-        $class = get_class($event);
-        $this->assertTrue($event == call_user_func([$class, 'deserialize'], $event->serialize()));
+        $serializedEvent = $event->serialize();
+        if ($event instanceof Forgettable) {
```

Comment:
we might want to consider adding a smoketest for events that must implement the Forgettable interface. Ensure that we don't make a mistake some time down the road.